clusters/hive/hypershift.yaml

```diff
@@ -113,3 +113,85 @@ subjects:
   - kind: ServiceAccount
     namespace: ci
     name: hypershift-workload
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  name: hypershift-worker
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machineset-role: hypershift
+  template:
+    metadata:
+      labels:
+        machineset-role: hypershift
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+    spec:
+      metadata:
+        labels:
+          hypershift.openshift.io/control-plane: 'true'
+      providerSpec:
+        value:
+          ami:
+            id: ami-03d1c2cba04df838c
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+            - ebs:
+                encrypted: true
+                iops: 100
+                kmsKey:
+                  arn: ''
+                volumeSize: 300
+                volumeType: gp2
+          securityGroups:
+            - filters:
+                - name: 'tag:Name'
+                  values:
+                    - hive-rj7rg-worker-sg
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: hive-rj7rg-worker-profile
+          instanceType: m5.4xlarge
```

Comment:
Do we really need a powerful type like this?

Reply:
The control plane pods are CPU intensive, so I use the same machine type as we run our e2e tests.

Reply:
Where do they run before this PR? Infra nodes?

Reply:
regular worker

Reply:
We had a discussion on the machineset over slack.
My concern is the cost on the more expensive machine types but you expect to reduce the number of of timeout which we do not have an effective way to monitor.
We could end up paying more for nothing.
I guess I should not block you further.